packages/renderer/src/dom/PDFDownloadLink.js

```diff
@@ -5,18 +5,17 @@ import React, { useEffect } from 'react';
 import usePDF from './usePDF';
 
 export const PDFDownloadLink = ({
-  style,
-  children,
-  className,
-  document: doc,
   fileName = 'document.pdf',
+  document,
```

Comment:
Let's keep alias so we avoid issues with global document value

```suggestion
  document: doc,
```